src/demands/demands.controller.ts

```diff
@@ -121,6 +136,12 @@ export class DemandsController {
         throw new AuthorizationError();
       }
 
+      this.journalsService.logInJournal({
```

Comment:
Inserts into `journals` table should be peformed in the same transaction as actions they log. Right now, every insert has its own transaction, so in this case here - we will end with journaling log stored in db even if deleting demand in line 145 fails.

Reply:
Idea if events is cool, so I am thinking - if we move all `logInJournal` calls in controllers outside the transaction we open, after `COMMIT`, then we are sure that everything is saved properly in db and we can propagate an event.

Example:
```typescript
  public async deleteSomething(id: string): Promise<void> {
    await this.sequelize.transaction(async (transaction) => {
      await this.somethingService.deleteSomething(transaction, id);
    });
    
    this.journalsService.logInJournal({...})
  }
```

Reply:
Will do as discussed on slack :) Thanks for this